src/auxinfo/auxinfo_commit.rs

```diff
@@ -123,22 +125,25 @@ impl CommitmentScheme {
         // Public parameters in this decommit must be consistent with each other
         scheme.clone().public_key.verify(context)?;
 
+        let lower_bound = BigNumber::one() << ELL;
+        assert!(scheme.public_key.params().scheme().modulus() >= &lower_bound);
+
```

Comment:
issue: This belongs to your other PR, it must have gotten ported here accidentally. Please delete it here.

Reply:
This still needs to be removed.